getting_started/intel_debug/introduction.rst

```diff
@@ -137,4 +137,36 @@ drivers and should work when changing the ``dsp_driver`` parameter.
 The selection of firmware, topology, and UCM files is based on platform
 capabilities, codec names, and DMI options. While the SOF team and the
 community try to cover all possible cases, errors will happen when the
-wrong file is selected at any of the three layers.
\ No newline at end of file
+wrong file is selected at any of the three layers.
+
+4. Chromebooks and SOF
+----------------------
+
+As stated above, Chromebook firmware is as of 2021 configured with the
+*community* key which means Chromebooks can run audio firmware signed by
+anyone. The entire filesystem is locked by default instead but there are
+a couple options to disable security for development purposes.
```

Comment:
recommend linking to dev mode process? https://chromium.googlesource.com/chromiumos/docs/+/master/developer_mode.md

Reply:
You're right, I didn't because I assume the developer guide below already would but the Developer Mode is of more general interest, not just for ChromeOS

Reply:
The link below is a few generations behind, this should always be up to date :)